tests/blanket-options.js

```diff
@@ -0,0 +1,19 @@
+/* globals blanket, module */
+
+var options = {
+  modulePrefix: 'ember-frost-notifier',
+  filter: '//.*ember-frost-notifier/.*/',
+  antifilter: '//.*(tests|template).*/',
+  loaderExclusions: [],
+  enableCoverage: true,
+  cliOptions: {
```

Comment:
Ah, here's why there's no coverage info. If you can update this to be like: https://github.com/ciena-frost/ember-frost-button/blob/master/tests/blanket-options.js#L9-L21 I think we can resolve #2